Chat/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+module.exports = (env) => {
+  const babelConfig = require('./.babelrc.js');
+  const commonConfig = require('./common.webpack.config')(__dirname, env, babelConfig);
```

Comment:
nothing shares this common.webpack.config - consider just having a single webpack file with all the info instead of extending common

Reply:
It will be easier to migrate in changes from the UI Library so lets keep this in for now.